### DISCUSSION TITLE
월드 갤럭시 호환

### DISCUSSION THREAD

**Discussion by maskpop** [naverz/zepeto-studio-kor#743] [5. Q&A]
안녕하세요 맵을 제작 후에 가이드에 따라서 파일등록까지 마치고 테스트해보려고 합니다. 테스터에 아이디를 넣고 들어가려하는데 아이폰으로는 들어갈 수 있지만 갤럭시로는 들어가지지 않았습니다. 혹시 이 문제에 해결법이 있나요??
버전의 문제가 있을 수도 있나요??

**Comment by x4cf** [comment]
접속에 실패하나요?

**Reply by maskpop** [reply]
제페토와 연결이 안되게 그냥 누를수가 없습니다...
![image](https://user-images.githubusercontent.com/105036393/168063868-35e1f738-2ed8-4e08-8d7d-7c608e6df298.png)
이런식입니다
직접 복사해서 인터넷에넣어봤는데도 나오지는 않습니다

**Reply by x4cf** [reply]
:D
https://github.com/naverz/zepeto-studio-kor/discussions/155

**Reply by maskpop** [reply]
감사합니다!!
